sdk/appconfiguration/perf-tests/app-configuration/test/listSettings.spec.ts

```diff
@@ -40,12 +43,13 @@ export class ListSettingsTest extends AppConfigTest<ListTestOptions> {
     for await (const response of this.client
       .listConfigurationSettings({ keyFilter: ListSettingsTest.prefix + "*" })
       .byPage()) {
+        // eslint-disable-next-line no-empty
```

Comment:
choosing to do this because this is what Harsha has in all the other perf tests

Reply:
I do not like this tbh but this is a comment to @HarshaNalluru. I always advocate for perf tests has meaningful logic. cc @mikeharder

Reply:
This is what we have in other languages as well.
The intention is that we don't do any additional work and to keep the list as minimal as possible which is ideal for perf testing.